src/main/webapp/app/layouts/search/results.html

```diff
@@ -0,0 +1,30 @@
+<div class="row">
+		<h3 class="tagline" align="centre"> Showing {{vm.users.length}} results</h3> 
+</div>
+
+<div class="container-fluid">
+	<!--div class="col-md-2"> </div-->
+	<div class="col-md-8 col-md-offset-2"> 
+			<div class="card" ng-repeat="user in vm.users">
+			  <img class="card-img-left" src="../../content/images/logo-jhipster.png" alt="Card image cap">
+			  <div class="card-block">
+			    <h4 class="card-title">{{user.name}}</h4>
+			    <p class="card-text"> Description Description Description Description Description Description Description Description </p>
```

Comment:
Test data to check how it resolves on the front end. Again, would be removed when the endpoint call is implemented - probably the next story for me. 

Shall I remove?